### PR TITLE
You can now safely start a subsystem by vv'ing can_fire to 1

### PR DESCRIPTION
**Pull request by MrsTonedOne**

### CODE REVIEW

**Comment by tkdrg** [comment]
Very neat :+1:

File: code/controllers/subsystem.dm [hunk 1]
```diff
@@ -73,4 +73,10 @@
 
 //usually called via datum/subsystem/New() when replacing a subsystem (i.e. due to a recurring crash)
 //should attempt to salvage what it can from the old instance of subsystem
-/datum/subsystem/proc/Recover()
\ No newline at end of file
+/datum/subsystem/proc/Recover()
+
+//this is so the subsystem doesn't rapid fire to make up missed ticks causing more lag
+/datum/subsytem/on_varedit(editedvar)
+	if (edited_var == can_fire && can_fire)
```
Comment (YotaXP) [inline code/controllers/subsystem.dm]: Is the first `can_fire` here not supposed to be in quotes?

Comment (YotaXP) [inline code/controllers/subsystem.dm]: You also misspelled `subsystem`.  Do you test your stuff? :cry:

Comment (ike709) [inline code/controllers/subsystem.dm]: @YotaXP Of course not, took him three tries to fix library computers :wink:

Comment (MrStonedOne) [inline code/controllers/subsystem.dm]: the branch name starts with patch, that means github editor, that means no, no testing 

=P

File: code/controllers/subsystem.dm [hunk 2]
```diff
@@ -73,4 +73,10 @@
 
 //usually called via datum/subsystem/New() when replacing a subsystem (i.e. due to a recurring crash)
 //should attempt to salvage what it can from the old instance of subsystem
-/datum/subsystem/proc/Recover()
\ No newline at end of file
+/datum/subsystem/proc/Recover()
+
+//this is so the subsystem doesn't rapid fire to make up missed ticks causing more lag
+/datum/subsystem/on_varedit(editedvar)
```
Comment (tkdrg) [inline code/controllers/subsystem.dm]: `edited_var`

**Comment by MrStonedOne** [comment]
3 line change, 4 commits.

**Pull request merged**